### PR TITLE
fix: aigrija — 2 stories

### PR DESCRIPTION
**Pull request by zenprocess**
## Summary
- **S-001**: Create Sanity Studio with schemas and integrate into admin Worker

## Test plan
- [x] `npx vitest run` — all tests passed
- [x] `npx tsc --noEmit` — clean
- [ ] Greptile review with zero unresolved comments

<h3>Greptile Summary</h3>

This PR introduces Sanity Studio (S-001) — seven new content schemas, a full Studio configuration, a seed script for initial Romanian content, a new SSR blog-page template with Portable Text rendering, and blog route families for all content categories in the Worker. The architecture is sound, but three issues must be resolved before production deployment:

- **ADMIN_CSP breaks Sanity Studio API connectivity** (`src/worker/admin/index.ts`): the `connect-src 'self'` directive in `ADMIN_CSP` is applied to every `text/html` response on the admin sub-router, including `studio/index.html`. The Studio must reach `*.api.sanity.io` and `cdn.sanity.io`; with the current policy those requests will be silently blocked by the browser.
- **Webhook endpoint has insecure / bypassable auth** (`src/worker/routes/blog.ts`): when `SANITY_WEBHOOK_SECRET` is not configured the auth block is skipped entirely, leaving the cache-invalidation endpoint fully open. Even when the secret is set, the check uses `.includes(secret)` instead of an HMAC comparison.
- **RSS feed routes missing for `rapoarte`, `povesti`, and `presa`** (`src/worker/routes/blog.ts`): the footer template renders a `/{category}/feed.xml` link for every category, but feed handlers are only registered for `amenintari`, `ghid`, and `educatie` — resulting in 404s for the other three.

All other aspects of the PR (schemas, SSR template, tests) are well-structured and production-ready.

<h3>Confidence Score: 2/5</h3>

- Not safe to merge — the Sanity Studio will be non-functional in production due to CSP misconfiguration, and the webhook endpoint has authentication gaps.
- Two of the three issues are likely to surface immediately in production: the Studio will fail silently (CSP blocks all Sanity API calls) and the webhook endpoint either skips auth entirely or uses an insecure check. These affect the core functionality introduced by this PR. The missing RSS feed routes cause 404s for three categories. None of these are edge cases — they impact primary features.
- src/worker/admin/index.ts (CSP + Studio integration) and src/worker/routes/blog.ts (webhook auth and missing RSS routes) need attention before merging.

<h3>Important Files Changed</h3>

| Filename | Overview |
|----------|----------|
| src/worker/admin/index.ts | Adds Sanity Studio SPA routes at /studio and /studio/*. Critical issue: the existing ADMIN_CSP middleware (connect-src 'self') is applied to all routes including studio/index.html, which will block the Studio from reaching any *.sanity.io API endpoints. This prevents the Studio from loading or saving content. |
| src/worker/routes/blog.ts | New blog routes for all content types (amenintari, ghid, educatie, rapoarte, povesti, presa) with list, detail, and RSS feed endpoints. Two issues: (1) webhook endpoint allows unauthenticated access when SANITY_WEBHOOK_SECRET is unset and uses insecure .includes() check instead of HMAC; (2) RSS feed routes are missing for rapoarte, povesti, and presa, but the footer template links to those routes for all categories causing 404s. |
| studio/sanity.config.ts | New Sanity Studio configuration with structured sidebar covering all content types. Uses a 'REPLACE_ME' fallback for projectId if env var is unset — safe as long as the build pipeline always provides SANITY_STUDIO_PROJECT_ID. |
| src/worker/templates/blog-page.ts | New SSR template producing full HTML pages for blog list and detail views, with Portable Text rendering, SEO meta tags, JSON-LD, and pagination. Solid implementation; note that footer RSS link covers all categories including those without registered feed routes. |
| src/worker/templates/blog-page.test.ts | Comprehensive test suite for the blog-page SSR template covering list and post pages, Portable Text rendering, SEO tags, and edge cases. Good coverage. |
| scripts/seed-content.ts | New seed script for populating Sanity with initial Romanian content (1 author, 3 categories, 4 blog posts, 2 threat reports, 1 weekly digest). Prior review comments on plain-string body fields and missing validation-level fields were acknowledged by the team and accepted as intentional (one-off seed script, backfillable via Studio). |

</details>

<details><summary><h3>Comments Outside Diff (4)</h3></summary>

1. `src/worker/admin/index.ts`, line 29 ([link](https://github.com/zenprocess/aigrija/blob/986c158b02e5acc45094abe1b373397707784ca1/src/worker/admin/index.ts#L29)) 

   **ADMIN_CSP breaks Sanity Studio**

   The global CSP middleware on this line applies to all admin routes, including the new `/studio` and `/studio/*` routes (lines 224–238). When `index.html` is served for the Studio, it receives the `ADMIN_CSP` header which includes `connect-src 'self'`. Sanity Studio needs to reach `https://<projectId>.api.sanity.io` and `https://<projectId>.apicdn.sanity.io` — these will be blocked by the `'self'`-only policy, making the Studio completely non-functional at runtime.

   **Fix:** Exempt the `/studio` and `/studio/*` routes from the CSP middleware and apply a Sanity-specific policy that includes:
   ```
   connect-src 'self' https://*.api.sanity.io https://*.apicdn.sanity.io wss://*.api.sanity.io
   ```

   You can add a dedicated middleware route before the global one:
   ```ts
   admin.use('/studio/*', cspHtmlMiddleware(STUDIO_CSP));
   admin.use('*', cspHtmlMiddleware(ADMIN_CSP));
   ```

   <details><summary>Prompt To Fix With AI</summary>

   `````markdown
   This is a comment left during a code review.
   Path: src/worker/admin/index.ts
   Line: 29

   Comment:
   **ADMIN_CSP breaks Sanity Studio**

   The global CSP middleware on this line applies to all admin routes, including the new `/studio` and `/studio/*` routes (lines 224–238). When `index.html` is served for the Studio, it receives the `ADMIN_CSP` header which includes `connect-src 'self'`. Sanity Studio needs to reach `https://<projectId>.api.sanity.io` and `https://<projectId>.apicdn.sanity.io` — these will be blocked by the `'self'`-only policy, making the Studio completely non-functional at runtime.

   **Fix:** Exempt the `/studio` and `/studio/*` routes from the CSP middleware and apply a Sanity-specific policy that includes:
   ```
   connect-src 'self' https://*.api.sanity.io https://*.apicdn.sanity.io wss://*.api.sanity.io
   ```

   You can add a dedicated middleware route before the global one:
   ```ts
   admin.use('/studio/*', cspHtmlMiddleware(STUDIO_CSP));
   admin.use('*', cspHtmlMiddleware(ADMIN_CSP));
   ```

   How can I resolve this? If you propose a fix, please make it concise.
   `````
   </details>

   <a href="https://app.greptile.com/ide/claude-code?prompt=This%20is%20a%20comment%20left%20during%20a%20code%20review.%0APath%3A%20src%2Fworker%2Fadmin%2Findex.ts%0ALine%3A%2029%0A%0AComment%3A%0A**ADMIN_CSP%20breaks%20Sanity%20Studio**%0A%0AThe%20global%20CSP%20middleware%20on%20this%20line%20applies%20to%20all%20admin%20routes%2C%20including%20the%20new%20%60%2Fstudio%60%20and%20%60%2Fstudio%2F*%60%20routes%20%28lines%20224%E2%80%93238%29.%20When%20%60index.html%60%20is%20served%20for%20the%20Studio%2C%20it%20receives%20the%20%60ADMIN_CSP%60%20header%20which%20includes%20%60connect-src%20'self'%60.%20Sanity%20Studio%20needs%20to%20reach%20%60https%3A%2F%2F%3CprojectId%3E.api.sanity.io%60%20and%20%60https%3A%2F%2F%3CprojectId%3E.apicdn.sanity.io%60%20%E2%80%94%20these%20will%20be%20blocked%20by%20the%20%60'self'%60-only%20policy%2C%20making%20the%20Studio%20completely%20non-functional%20at%20runtime.%0A%0A**Fix%3A**%20Exempt%20the%20%60%2Fstudio%60%20and%20%60%2Fstudio%2F*%60%20routes%20from%20the%20CSP%20middleware%20and%20apply%20a%20Sanity-specific%20policy%20that%20includes%3A%0A%60%60%60%0Aconnect-src%20'self'%20https%3A%2F%2F*.api.sanity.io%20https%3A%2F%2F*.apicdn.sanity.io%20wss%3A%2F%2F*.api.sanity.io%0A%60%60%60%0A%0AYou%20can%20add%20a%20dedicated%20middleware%20route%20before%20the%20global%20one%3A%0A%60%60%60ts%0Aadmin.use%28'%2Fstudio%2F*'%2C%20cspHtmlMiddleware%28STUDIO_CSP%29%29%3B%0Aadmin.use%28'*'%2C%20cspHtmlMiddleware%28ADMIN_CSP%29%29%3B%0A%60%60%60%0A%0AHow%20can%20I%20resolve%20this%3F%20If%20you%20propose%20a%20fix%2C%20please%20make%20it%20concise.&repo=zenprocess%2Faigrija"><picture><source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/badge/Fix_in_Claude-black?logo=claude&logoColor=%23D97706">Fix in Claude Code</picture></a>

2. `src/worker/admin/index.ts`, line 29 ([link](https://github.com/zenprocess/aigrija/blob/d6bd5f8037295629cfcf64ed3554c69e6c1ad302/src/worker/admin/index.ts#L29)) 

   **ADMIN_CSP breaks Sanity Studio API connectivity**

   `ADMIN_CSP` contains `connect-src 'self'` (see `src/worker/lib/csp.ts` line 19), which restricts all `XMLHttpRequest`/`fetch` calls to the same origin. The Sanity Studio served at `/admin/studio` must connect to external Sanity APIs (`https://{projectId}.api.sanity.io`, `https://cdn.sanity.io`, etc.) to load documents, run mutations, and stream real-time updates.

   Because `cspHtmlMiddleware` applies the policy to every `text/html` response — including `studio/index.html` served at lines 224–237 — the browser will block every outbound request the Studio makes, causing it to appear to load but silently fail to fetch or save any content.

   **Fix:** Apply a dedicated, more-permissive CSP for the studio routes before mounting the wildcard middleware:

   ```ts
   // Add before line 29:
   admin.use('/studio/*', cspHtmlMiddleware(
     "default-src 'self'; script-src 'self' 'unsafe-inline' https://*.sanity.io https://cdn.sanity.io; style-src 'self' 'unsafe-inline'; img-src 'self' data: https:; connect-src 'self' https://*.sanity.io https://cdn.sanity.io; frame-ancestors 'none';"
   ));

   // Then keep the existing line:
   admin.use('*', cspHtmlMiddleware(ADMIN_CSP));
   ```

3. `src/worker/routes/blog.ts`, line 644-650 ([link](https://github.com/zenprocess/aigrija/blob/d6bd5f8037295629cfcf64ed3554c69e6c1ad302/src/worker/routes/blog.ts#L644-L650)) 

   **Webhook auth is skipped entirely when secret is unconfigured, and the check is not HMAC-based**

   There are two distinct problems here:

   1. **Auth is opt-in** — the entire validation block is guarded by `if (c.env.SANITY_WEBHOOK_SECRET)`. If the secret is not set (e.g., in a staging environment or after a misconfigured deployment), any unauthenticated `POST` to this endpoint triggers a full KV cache purge (lines 652–666). While this is a DoS/cache-thrashing vector rather than a data-corruption risk, it leaves the endpoint fully open. Return `401` immediately if the secret is absent rather than skipping validation.

   2. **`.includes()` is not a valid signature check** — Sanity sends an HMAC-SHA256 digest in the `sanity-webhook-signature` header, not the raw secret. The `.includes(secret)` check (line 647) verifies only that the header value contains the secret as a substring, which is trivially satisfiable by an attacker who can enumerate short secrets. Compute `HMAC-SHA256(body, secret)` and compare it (in constant-time) to the provided digest.

   **Fix:**

   ```ts
   blog.post('/content/webhook', async (c) => {
     if (!c.env.SANITY_WEBHOOK_SECRET) {
       return c.json({ error: { code: 'UNAUTHORIZED', message: 'Webhook not configured.' }, request_id: 'unknown' }, 401);
     }

     const signature = c.req.header('sanity-webhook-signature') ?? '';
     const body = await c.req.text();
     
     // Compute HMAC-SHA256
     const encoder = new TextEncoder();
     const keyData = encoder.encode(c.env.SANITY_WEBHOOK_SECRET);
     const messageData = encoder.encode(body);
     const key = await crypto.subtle.importKey('raw', keyData, { name: 'HMAC', hash: 'SHA-256' }, false, ['sign']);
     const digest = await crypto.subtle.sign('HMAC', key, messageData);
     const computed = Array.from(new Uint8Array(digest)).map(b => b.toString(16).padStart(2, '0')).join('');
     
     // Constant-time comparison
     if (computed !== signature.replace('sha256=', '')) {
       return c.json({ error: { code: 'UNAUTHORIZED', message: 'Semnatura invalida.' }, request_id: 'unknown' }, 401);
     }

     try {
       // ... rest of handler (lines 652–666)
     } catch (err) {
       // ...
     }
   });
   ```

4. `src/worker/routes/blog.ts`, line 352-408 ([link](https://github.com/zenprocess/aigrija/blob/d6bd5f8037295629cfcf64ed3554c69e6c1ad302/src/worker/routes/blog.ts#L352-L408)) 

   **Missing RSS feed routes for `rapoarte`, `povesti`, and `presa`**

   The `renderFooter()` helper in `src/worker/templates/blog-page.ts` (line 94) unconditionally renders an RSS feed link for every category:

   ```html
   <a href="/${category}/feed.xml">RSS Feed</a>
   ```

   RSS feed routes are only registered for `amenintari` (line 148), `ghid` (line 226), and `educatie` (line 304). The `rapoarte`, `povesti`, and `presa` sections define list and detail routes (lines 354–408, 412–466, 470–524) but have no corresponding `feed.xml` handler, so clicking "RSS Feed" in the footer on any of those pages returns a 404.

   **Fix:** Add feed route handlers for `/rapoarte/feed.xml`, `/povesti/feed.xml`, and `/presa/feed.xml` following the same pattern as the existing feed routes (e.g., lines 226–244 for ghid). Each should query the appropriate content type (`weeklyDigest`, `communityStory`, `pressRelease`) and use `buildRss()` helper to generate the XML response.

</details>

<a href="https://app.greptile.com/ide/claude-code?prompt=Fix%20the%20following%203%20code%20review%20issues.%20Work%20through%20them%20one%20at%20a%20time%2C%20proposing%20concise%20fixes.%0A%0A---%0A%0A%23%23%23%20Issue%201%20of%203%0Asrc%2Fworker%2Fadmin%2Findex.ts%3A29%0A**ADMIN_CSP%20breaks%20Sanity%20Studio%20API%20connectivity**%0A%0A%60ADMIN_CSP%60%20contains%20%60connect-src%20'self'%60%20%28see%20%60src%2Fworker%2Flib%2Fcsp.ts%60%20line%2019%29%2C%20which%20restricts%20all%20%60XMLHttpRequest%60%2F%60fetch%60%20calls%20to%20the%20same%20origin.%20The%20Sanity%20Studio%20served%20at%20%60%2Fadmin%2Fstudio%60%20must%20connect%20to%20external%20Sanity%20APIs%20%28%60https%3A%2F%2F%7BprojectId%7D.api.sanity.io%60%2C%20%60https%3A%2F%2Fcdn.sanity.io%60%2C%20etc.%29%20to%20load%20documents%2C%20run%20mutations%2C%20and%20stream%20real-time%20updates.%0A%0ABecause%20%60cspHtmlMiddleware%60%20applies%20the%20policy%20to%20every%20%60text%2Fhtml%60%20response%20%E2%80%94%20including%20%60studio%2Findex.html%60%20served%20at%20lines%20224%E2%80%93237%20%E2%80%94%20the%20browser%20will%20block%20every%20outbound%20request%20the%20Studio%20makes%2C%20causing%20it%20to%20appear%20to%20load%20but%20silently%20fail%20to%20fetch%20or%20save%20any%20content.%0A%0A**Fix%3A**%20Apply%20a%20dedicated%2C%20more-permissive%20CSP%20for%20the%20studio%20routes%20before%20mounting%20the%20wildcard%20middleware%3A%0A%0A%60%60%60ts%0A%2F%2F%20Add%20before%20line%2029%3A%0Aadmin.use%28'%2Fstudio%2F*'%2C%20cspHtmlMiddleware%28%0A%20%20%22default-src%20'self'%3B%20script-src%20'self'%20'unsafe-inline'%20https%3A%2F%2F*.sanity.io%20https%3A%2F%2Fcdn.sanity.io%3B%20style-src%20'self'%20'unsafe-inline'%3B%20img-src%20'self'%20data%3A%20https%3A%3B%20connect-src%20'self'%20https%3A%2F%2F*.sanity.io%20https%3A%2F%2Fcdn.sanity.io%3B%20frame-ancestors%20'none'%3B%22%0A%29%29%3B%0A%0A%2F%2F%20Then%20keep%20the%20existing%20line%3A%0Aadmin.use%28'*'%2C%20cspHtmlMiddleware%28ADMIN_CSP%29%29%3B%0A%60%60%60%0A%0A%23%23%23%20Issue%202%20of%203%0Asrc%2Fworker%2Froutes%2Fblog.ts%3A644-650%0A**Webhook%20auth%20is%20skipped%20entirely%20when%20secret%20is%20unconfigured%2C%20and%20the%20check%20is%20not%20HMAC-based**%0A%0AThere%20are%20two%20distinct%20problems%20here%3A%0A%0A1.%20**Auth%20is%20opt-in**%20%E2%80%94%20the%20entire%20validation%20block%20is%20guarded%20by%20%60if%20%28c.env.SANITY_WEBHOOK_SECRET%29%60.%20If%20the%20secret%20is%20not%20set%20%28e.g.%2C%20in%20a%20staging%20environment%20or%20after%20a%20misconfigured%20deployment%29%2C%20any%20unauthenticated%20%60POST%60%20to%20this%20endpoint%20triggers%20a%20full%20KV%20cache%20purge%20%28lines%20652%E2%80%93666%29.%20While%20this%20is%20a%20DoS%2Fcache-thrashing%20vector%20rather%20than%20a%20data-corruption%20risk%2C%20it%20leaves%20the%20endpoint%20fully%20open.%20Return%20%60401%60%20immediately%20if%20the%20secret%20is%20absent%20rather%20than%20skipping%20validation.%0A%0A2.%20**%60.includes%28%29%60%20is%20not%20a%20valid%20signature%20check**%20%E2%80%94%20Sanity%20sends%20an%20HMAC-SHA256%20digest%20in%20the%20%60sanity-webhook-signature%60%20header%2C%20not%20the%20raw%20secret.%20The%20%60.includes%28secret%29%60%20check%20%28line%20647%29%20verifies%20only%20that%20the%20header%20value%20contains%20the%20secret%20as%20a%20substring%2C%20which%20is%20trivially%20satisfiable%20by%20an%20attacker%20who%20can%20enumerate%20short%20secrets.%20Compute%20%60HMAC-SHA256%28body%2C%20secret%29%60%20and%20compare%20it%20%28in%20constant-time%29%20to%20the%20provided%20digest.%0A%0A**Fix%3A**%0A%0A%60%60%60ts%0Ablog.post%28'%2Fcontent%2Fwebhook'%2C%20async%20%28c%29%20%3D%3E%20%7B%0A%20%20if%20%28!c.env.SANITY_WEBHOOK_SECRET%29%20%7B%0A%20%20%20%20return%20c.json%28%7B%20error%3A%20%7B%20code%3A%20'UNAUTHORIZED'%2C%20message%3A%20'Webhook%20not%20configured.'%20%7D%2C%20request_id%3A%20'unknown'%20%7D%2C%20401%29%3B%0A%20%20%7D%0A%0A%20%20const%20signature%20%3D%20c.req.header%28'sanity-webhook-signature'%29%20%3F%3F%20''%3B%0A%20%20const%20body%20%3D%20await%20c.req.text%28%29%3B%0A%20%20%0A%20%20%2F%2F%20Compute%20HMAC-SHA256%0A%20%20const%20encoder%20%3D%20new%20TextEncoder%28%29%3B%0A%20%20const%20keyData%20%3D%20encoder.encode%28c.env.SANITY_WEBHOOK_SECRET%29%3B%0A%20%20const%20messageData%20%3D%20encoder.encode%28body%29%3B%0A%20%20const%20key%20%3D%20await%20crypto.subtle.importKey%28'raw'%2C%20keyData%2C%20%7B%20name%3A%20'HMAC'%2C%20hash%3A%20'SHA-256'%20%7D%2C%20false%2C%20%5B'sign'%5D%29%3B%0A%20%20const%20digest%20%3D%20await%20crypto.subtle.sign%28'HMAC'%2C%20key%2C%20messageData%29%3B%0A%20%20const%20computed%20%3D%20Array.from%28new%20Uint8Array%28digest%29%29.map%28b%20%3D%3E%20b.toString%2816%29.padStart%282%2C%20'0'%29%29.join%28''%29%3B%0A%20%20%0A%20%20%2F%2F%20Constant-time%20comparison%0A%20%20if%20%28computed%20!%3D%3D%20signature.replace%28'sha256%3D'%2C%20''%29%29%20%7B%0A%20%20%20%20return%20c.json%28%7B%20error%3A%20%7B%20code%3A%20'UNAUTHORIZED'%2C%20message%3A%20'Semnatura%20invalida.'%20%7D%2C%20request_id%3A%20'unknown'%20%7D%2C%20401%29%3B%0A%20%20%7D%0A%0A%20%20try%20%7B%0A%20%20%20%20%2F%2F%20...%20rest%20of%20handler%20%28lines%20652%E2%80%93666%29%0A%20%20%7D%20catch%20%28err%29%20%7B%0A%20%20%20%20%2F%2F%20...%0A%20%20%7D%0A%7D%29%3B%0A%60%60%60%0A%0A%23%23%23%20Issue%203%20of%203%0Asrc%2Fworker%2Froutes%2Fblog.ts%3A352-408%0A**Missing%20RSS%20feed%20routes%20for%20%60rapoarte%60%2C%20%60povesti%60%2C%20and%20%60presa%60**%0A%0AThe%20%60renderFooter%28%29%60%20helper%20in%20%60src%2Fworker%2Ftemplates%2Fblog-page.ts%60%20%28line%2094%29%20unconditionally%20renders%20an%20RSS%20feed%20link%20for%20every%20category%3A%0A%0A%60%60%60html%0A%3Ca%20href%3D%22%2F%24%7Bcategory%7D%2Ffeed.xml%22%3ERSS%20Feed%3C%2Fa%3E%0A%60%60%60%0A%0ARSS%20feed%20routes%20are%20only%20registered%20for%20%60amenintari%60%20%28line%20148%29%2C%20%60ghid%60%20%28line%20226%29%2C%20and%20%60educatie%60%20%28line%20304%29.%20The%20%60rapoarte%60%2C%20%60povesti%60%2C%20and%20%60presa%60%20sections%20define%20list%20and%20detail%20routes%20%28lines%20354%E2%80%93408%2C%20412%E2%80%93466%2C%20470%E2%80%93524%29%20but%20have%20no%20corresponding%20%60feed.xml%60%20handler%2C%20so%20clicking%20%22RSS%20Feed%22%20in%20the%20footer%20on%20any%20of%20those%20pages%20returns%20a%20404.%0A%0A**Fix%3A**%20Add%20feed%20route%20handlers%20for%20%60%2Frapoarte%2Ffeed.xml%60%2C%20%60%2Fpovesti%2Ffeed.xml%60%2C%20and%20%60%2Fpresa%2Ffeed.xml%60%20following%20the%20same%20pattern%20as%20the%20existing%20feed%20routes%20%28e.g.%2C%20lines%20226%E2%80%93244%20for%20ghid%29.%20Each%20should%20query%20the%20appropriate%20content%20type%20%28%60weeklyDigest%60%2C%20%60communityStory%60%2C%20%60pressRelease%60%29%20and%20use%20%60buildRss%28%29%60%20helper%20to%20generate%20the%20XML%20response.%0A%0A&repo=zenprocess%2Faigrija"><picture><source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/badge/Fix_All_in_Claude-black?logo=claude&logoColor=%23D97706">Fix All in Claude Code</picture></a>

<sub>Last reviewed commit: d6bd5f8</sub>

### CODE REVIEW

**Comment by zenprocess** [comment]
@greptile-ai please review

File: scripts/seed-content.ts [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+Campania exploatează obișnuința românilor de a primi facturi electronice de la eMAG. E-mailurile sunt bine construite, conțin logo-ul oficial, un număr de comandă plauzibil și o sumă de plată realistă. Atașamentul PDF conține cod JavaScript obfuscat care descarcă payload-ul malițios de pe servere compromise din Turcia și Ucraina.
+
+Indicatorii de compromitere (IoC) au fost partajați cu companiile de antivirus și cu CERT-RO. Recomandăm utilizatorilor să nu deschidă atașamente din e-mailuri nesolicitate, să verifice comenzile direct pe site-ul emag.ro și să mențină antivirusul actualizat. eMAG a confirmat că facturile oficiale sunt disponibile exclusiv în contul de utilizator de pe platformă, nu prin e-mail.
+
+Dacă ați deschis un astfel de atașament, deconectați imediat dispozitivul de la internet, efectuați o scanare completă cu antivirusul, schimbați parolele conturilor bancare de pe un alt dispozitiv și contactați banca pentru monitorizarea tranzacțiilor suspecte.`,
+    firstSeen: '2026-03-05T14:30:00Z',
+    publishedAt: '2026-03-06T09:00:00Z',
+    language: 'ro',
+    categories: [{ _type: 'reference' as const, _ref: 'category-amenintari' }],
+  },
```
[the remark this answers is not in the document]
Comment (zenprocess) [reply scripts/seed-content.ts]: Acknowledged. initialValue only applies in Studio UI, not via mutations API. The threat reports display correctly without it. Can be set to 'active' via Studio.

**Comment by zenprocess** [comment]
@greptile-ai please review
